src/app/child-dev-project/schools/children-overview/children-overview.component.spec.ts

```diff
@@ -55,7 +55,8 @@ describe("ChildrenOverviewComponent", () => {
 
     expect(mockChildrenService.queryRelationsOf).toHaveBeenCalledWith(
       "school",
-      school.getId()
+      school.getId(),
+      true
```

Comment:
should we switch this param, so that the default (undefined or false) will result in the previous behaviour of excluding inactive ones?